git_repo/repo.py

```diff
@@ -163,15 +163,16 @@ def _guess_repo_slug(self, repository, service, resolve_targets=None):
         for remote in repository.remotes:
             if remote.name in targets:
                 for url in remote.urls:
-                    if url.startswith('https'):
-                        if url.endswith('.git'):
-                            url = url[:-4]
+                    if url.endswith('.git'):
+                        url = url[:-4]
+                    if url.startswith('https') or url.startswith('http'):
                         *_, user, name = url.split('/')
                         self.set_repo_slug('/'.join([user, name]))
                         return
+                    elif url.startswith('ssh://'):
```

Comment:
> Gogs can configure ssh username. (Of course it is rare case)

hmm... it means `startswith('git@')` doesn't match for Gogs...
In my Gogs instance running on Raspberry Pi, ssh URL is: `pi@raspberrypi.local:test/test.git`

But it is not GitBucket related fix. Which is better action?
Should I create new Issue/PR? or fix it in this PR?

It should be rewrite as:
```
                for url in remote.urls:
                    if url.endswith('.git'):
                        url = url[:-4]
                    if url.find('://') > -1:
                        *_, user, name = url.split('/')
                        self.set_repo_slug('/'.join([user, name]))
                        return
                    elif url.find('@') > -1:
                        _, repo_slug = url.split(':')
                        self.set_repo_slug(repo_slug)
                        return
```

but it is not tested.